view/frontend/web/js/telephoneValidatorAddress.js

```diff
@@ -33,12 +37,15 @@ define([
 
                 return isValid;
             }
-        }
+        };
```

Comment:
@zergionn really tiny thing, please can you de-indent this line to 4 spaces instead of 8?